README.md

```diff
@@ -4,23 +4,57 @@
 [![codecov](https://codecov.io/gh/ostis-ai/sc-machine/branch/main/graph/badge.svg?token=WU8O9Z1DNL)](https://codecov.io/gh/ostis-ai/sc-machine)
 [![docker](https://img.shields.io/docker/v/ostis/sc-machine?arch=amd64&label=Docker&logo=Docker&sort=date)](https://hub.docker.com/r/ostis/sc-machine)
 
-# Documentation: [GitHub Pages](https://ostis-ai.github.io/sc-machine)
+<b>SC-machine</b> is a software implementation of semantic network storage and processing of information in language
+of universal knowledge representation SC-code, provided by [<b>the OSTIS Technology</b>](https://github.com/ostis-ai/ostis-standard). 
+SC-machine is a <u>platform-independent graph database management system</u> to store, retrieves and run tasks (agents) 
+on knowledge graphs. All declarative (data, data structures, documentation, tasks specification, etc.) and procedural 
+(programs, modules, systems, communication between systems) knowledge represents on one language, named as SC-code. 
+Nowadays sc-machine includes:
+- semantic storage to store graph constructions, named as sc-memory (graph database);
+- event-based agents processing, named as sc-event-manager (asynced programs workflows handling);
+- tools to communicate with storage:
+   - <u>native</u> program API on C++;
+   - <u>network</u> program API, provided by <i>sc-server</i> to communicate with storage by network with help of Websocket 
+     and JSON languages;
+   - <i>sc-builder</i> to load information from SCs files into storage (SCs-code is a one of external representation 
+     languages of SC-code);
+
+SC-machine provide complete and sufficient native and network APIs to create, update, delete and retrieve operations and
+also supports event-driven communication with sc-memory and sc-agents, located in it.
+
+Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions <u>by the 
+way of one technology</u>, <u>one data representation and programming language</u> and <u>one program software</u>!
+
+# Documentation: 
+
+- A part of sc-machine documentation, represented in md format: [GitHub Pages](https://ostis-ai.github.io/sc-machine).
+  It serves as a guide for installation, development and a brief description of the project modules.
+- SCn-code represented sc-machine documentation: [LaTeX documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
+  This documentation fully describes the sc-machine: its components, software interfaces for interacting with sc-memory, 
+  the characteristics and design of sc-memory, as well as the rationale for the sc-machine tools and solutions used.
 
 # Quick start
+Sc-machine is a part of the whole system, named as <b>OSTIS Web platform</b>.
```

Comment:
What do you mean by "whole system"? I think it is better to say that sc-machine is a core for an ostis-system. OSTIS Web Platform is an example of one